modelle-718.css

```diff
@@ -19,18 +19,18 @@ p{
     padding: 0;
     text-align: center;
     font-family: "Istok Web";
-    font-size: 35px;
+    font-size: 3.24vh;
     font-style: normal;
     font-weight: 700;
     line-height: normal;
 }
 
 h1{
-    margin: 19px auto 30px auto;
+    margin: 0.926vh auto 2.7777vh auto;
```

Comment:
Hier würde ich auch eine andere Maßeinheit nutzen. Abstände erzeugen Leerraum ("whitespace"), der sich immer möglichst harmonisch zu den Inhalten verhalten sollte. Großer Text sieht besser mit großem Leerraum aus, kleiner Text wirkt bei großem Leerraum schnell verloren.

Richtig clever ist es daher, wenn man Abstände nicht relativ zur Bildschirmgröße, sondern zur Textgröße sieht. Wenn du Lust hast, dich in dieses schnell recht kompliziert werdende Thema einzulesen, ist https://utopia.fyi/ eine tolle Quelle.

Ein quick fix wäre hier, für die Abstände von `vh` auf `em` zu wechseln.